pyresample/test/test_geometry/test_area.py

```diff
@@ -0,0 +1,2003 @@
+# Copyright (C) 2010-2022 Pyresample developers
```

Comment:
Feels like this is a copy of existing tests? Is this really needed? can't we have just one version of this?

Reply:
"one version" of what? Of the AreaDefinition tests? Please read my comments with my questions and my plans to remove the legacy-only tests right after this PR is merged.

Reply:
yeah, this wasn't about the copyright :)
Ok, so I think it's a good idea to have have tests for both legacy and future in one module. But I would like to avoid being in a limbo for a while where we have duplicated tests. Do you think it's possible to remove the legacy tests already in this PR?

Reply:
If that's really what you want. The idea was that I was going to make a ton of PRs right away after this merge a week or so ago when I had time. Now I'm not so sure how fast I can do that so I guess I can put it here.